docs/zh/user_guide/ani_source_provider/README.md

```diff
@@ -6,33 +6,182 @@ ANi Project是一个全自动下载流媒体网站（[动画疯](https://ani.gam
 
 本项目将会同步ANi更新的所有番剧到你的私有库中。由于是内嵌字幕，该项目的视频源不适合用作收藏，我们推荐定期清除旧的下载。
 
-因为ANi的API提供的是直链下载，所以本源要求必须正确配置好aria2或其他类似下载软件。
+同时支持ANi的直链下载API以及Torrent API。
 
 ## 前提
 你已经安装好了Kubespider。
 
+如果使用Torrent API，我们强烈推荐在`download_provider.yaml`中设定qBittorrent的优先级高于Aria2，以使用qBittorrent的相关新特性。
```

Comment:
并非，目前所有的主流版本都支持子目录功能。
区别是，旧版本的子目录路径稍微有点bug，不太好用，摘录我之前向qbittorrent反馈的这个[issue](https://github.com/qbittorrent/qBittorrent/issues/19941)：

> currently, when i add a subcategory which wasn't exist before by using qbitorrent api, like:
>
> ```python
> if qbt_client.torrents_add(urls=magnet_3, save_path="C:/Users/ChowD/Pictures", category="TVs/The Boys", use_auto_torrent_management=True) != "Ok.":
>     raise Exception("Failed to add torrent.")
> ```
>
> 
>
> Because use_auto_torrent_management was set to be True, the save_path was forcebly covered, that was what i expected, no problem.
>
> If i specify an existed category, e.g. `TVs`, the new torrent will save into its defined path, lets call it `$category_save_path`, no problem.
>
> However, if i specify an existed category with a subcategory that doesnt exist, the subcategory will be created automatically, that's very ok, but the save path will turn into `$base_save_path/$category_name/$sub_category_name`
>
> I think there's a option for us to choose new subcategory auto torrent management save path to be:
>
> ```
> $base_save_path/$category_name/$sub_category_name
> or
> $category_save_path/$sub_category_name
> ```
>
> Apparently `$category_save_path/$sub_category_name` is more suitable for this scenario